tests/xstate/atomWithMachine.test.tsx

```diff
@@ -47,4 +64,10 @@ it('toggle machine', async () => {
 
   fireEvent.click(getByText('Active! Click to deactivate'))
   await findByText('Click to activate')
+
+  fireEvent.click(getByText('Go to final state'))
+  await findByText('Not actionable')
+
+  fireEvent.click(getByText('Restart machine'))
+  await waitFor(() => expect(findByText('Click to activate')).toBeDefined())
```

Comment:
cool. seems working.

can you please create a new spec, instead of modifying the existing one?
```js
it('resettable machine', () => {
```

Reply:
okay.